packages/terra-functional-testing/src/reporters/wdio-accessibility-reporter.js

```diff
@@ -0,0 +1,123 @@
+const chalk = require('chalk');
+const WDIOReporter = require('@wdio/reporter').default;
+
+class AccessibilityReporter extends WDIOReporter {
+  /**
+   * Indents a string.
+   * @param {string} string - The string to indent.
+   * @param {number} indent - The number of spaces to indent the string.
+   */
+  static indent(string, indent) {
+    if (indent > 0) {
+      return string.replace(/^(?!\s*$)/gm, Array(indent + 1).join(' '));
+    }
+
+    return string;
+  }
+
+  constructor(options) {
+    super(options);
+
+    this.currentTest = null;
+    this.accessibilityResults = {};
+
+    process.on('terra:report:accessibility', this.onReportAccessibility.bind(this));
+  }
+
+  /**
+   * Hook invoked at the start of the test.
+   * @param {TestStats} test - The current test object.
+   */
+  onTestStart(test) {
+    this.currentTest = test.uid;
+  }
+
+  /**
+   * Hook invoked when accessibility results are reported.
+   * @param {Object} results - Accessibility results.
+   * @param {array} results.incomplete - Incomplete accessibility results.
+   */
+  onReportAccessibility(results) {
+    this.accessibilityResults[this.currentTest] = results;
+  }
+
+  /**
+   * Hook invoked when the runner ends.
+   * @param RunnerStats runner - The test runner stats object.
+   */
+  onRunnerEnd(runner) {
+    this.printReport(runner);
+  }
+
+  /**
+   * Print the accessibility report.
+   * @param {RunnerStats} runner - The test runner.
+   */
+  printReport(runner) {
+    const warnings = Object.keys(this.accessibilityResults);
+
+    if (warnings.length === 0) {
+      return;
+    }
+
+    const rootSuite = this.currentSuites.find((suite) => suite.uid === '(root)');
+    const output = this.travelSuite(rootSuite);
+    const spec = runner.specs[0];
+
+    this.write(`Spec: ${spec}\n${output}`);
```

Comment:
Can we output this as a report as well that can be pulled in as an build artifact and as a release site artifact?